types/simple-oauth2/index.d.ts

```diff
@@ -113,7 +115,7 @@ export interface OAuthClient<ClientIdName extends string = 'client_id'> {
         ): string,
 
         /** Returns the Access Token object */
-        getToken(params: AuthorizationTokenConfig): Promise<Token>;
+        getToken(params: AuthorizationTokenConfig, httpOptions?: any): Promise<Token>;
```

Comment:
According to [this commit on simple-oauth2](https://github.com/lelylan/simple-oauth2/commit/beb0c316b4fc8feab835071d77646daaf95bc641), `ownerPassword`, `clientCredentials` and `accessToken` also got this parameter added.

Reply:
Also, what types can `httpOptions` be?

Reply:
Ok, added the param to the other functions

The type seems to be an http options object from the Wreck library:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/wreck/index.d.ts#L17-L41

I've copied the interface over